src/less/components/_data.less

```diff
@@ -63,7 +63,17 @@
 // Markup:
 //<p class="statusDataValue">
 //  <span class="active">
-//    Active
+//    Active Label
+//  </span>
+//</p> <br />
+//<p class="statusDataValue">
```

Comment:
So I have replaced the statusDataValue with bootstrap label in the card examples and and also the Status Value Label. Are we all set here?

@jensru @taifunhaeberle